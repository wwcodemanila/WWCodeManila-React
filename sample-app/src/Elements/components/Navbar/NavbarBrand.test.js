import React from "react";
import { shallow } from 'enzyme';

import NavbarBrand from './NavbarBrand'

test('test name', () => {
  // test case
  const component = shallow(<NavbarBrand />)
  expect(component).toMatchSnapshot()
})
