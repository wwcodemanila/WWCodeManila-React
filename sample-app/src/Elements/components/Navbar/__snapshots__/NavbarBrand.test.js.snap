// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test name 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavbarBrand />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <img
        alt="Women Who Code Manila"
        className="logo"
        src="wwcode.png"
      />,
      "className": "navbar-brand",
      "replace": false,
      "to": "/",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "alt": "Women Who Code Manila",
        "className": "logo",
        "src": "wwcode.png",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <img
          alt="Women Who Code Manila"
          className="logo"
          src="wwcode.png"
        />,
        "className": "navbar-brand",
        "replace": false,
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Women Who Code Manila",
          "className": "logo",
          "src": "wwcode.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
